### ISSUE TITLE
Create project to enable AnnotationRule

### ISSUE THREAD

**Issue by uziassantosferreira** [pinterest/ktlint#502]
I created a project to enable AnnotationRule() and when I run and I get this error 
`Internal Error (custom-ktlint-rules:annotation). Please create a ticket at https://github.com/shyiko/ktlint/issue (if possible, provide the source code that triggered an error) (cannot be auto-corrected)
`

My custom RuleSet 
```
import com.pinterest.ktlint.core.RuleSet
import com.pinterest.ktlint.core.RuleSetProvider
import com.pinterest.ktlint.ruleset.experimental.ImportOrderingRule
import com.pinterest.ktlint.ruleset.standard.AnnotationRule
import com.pinterest.ktlint.ruleset.standard.IndentationRule
import com.pinterest.ktlint.ruleset.standard.MultiLineIfElseRule
import com.pinterest.ktlint.ruleset.standard.NoWildcardImportsRule

class CustomRuleSetProvider : RuleSetProvider {
    override fun get() = RuleSet("custom-ktlint-rules",
        NoWildcardImportsRule(),
        MultiLineIfElseRule(),
        ImportOrderingRule(),
        AnnotationRule(),
        IndentationRule())
}

```

When I remove `AnnotationRule` the ktlint works normal

**Comment by shashachu** [comment]
The AnnotationRule was disabled in the standard ruleset due to errors; it may be fixed by https://github.com/pinterest/ktlint/pull/497

**Comment by shashachu** [comment]
PR #509 fixes this. It will be released in 0.34.0
